internal/datacoord/garbage_collector.go

```diff
@@ -153,6 +157,7 @@ func (gc *garbageCollector) scan() {
 			)
 		}
 		segmentMap, filesMap := getMetaMap()
+		metrics.GarbageCollectorListLatency.WithLabelValues(fmt.Sprint(paramtable.GetNodeID()), labels[idx])
```

Comment:
Here we'd call the `#Observe(duration)` to add a observe.

Reply:
my fault, fixing it